### PR TITLE
Added recipe for shopware deployments.

### PR DESCRIPTION
**Pull request by arvatis**

### CODE REVIEW

**Comment by oanhnn** [comment]
@arvatis We need more information about this PR.

**Comment by arvatis** [comment]
Which kind of information ? 
Its a set of recipes in order to be able to deploy shopware :)

File: docs/shopware.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Cachetool recipe
```
Comment (elfet) [inline docs/shopware.md]: Chachetool?

**Comment by elfet** [comment]
It's better to add this king of recipe to main repo.

**Comment by elfet** [comment]
Please, rebase your PR.

**Pull request closed without merging**